### ISSUE TITLE
Maven build error trying to use version 0.3 as maven dependency

### ISSUE THREAD

**Issue by DarioRomano** [Universal-Variability-Language/uvl-parser#40]
Hi, I'm currently trying to build the Travart core from this branch using maven: https://github.com/SECPS/TraVarT/tree/updated-parser-dependency

I changed the parser dependency to :
```
<dependency>
            <groupId>io.github.universal-variability-language</groupId>
	    <artifactId>uvl-parser</artifactId>
	    <version>0.3</version>
</dependency>
```

unfortunately I get errors that all the described packages do not exist. Any clue what could be causing this issue?

![image](https://github.com/Universal-Variability-Language/uvl-parser/assets/19183169/cf94255b-84c1-493d-989e-d6803b0a44ae)

Cheers

**Comment by SundermannC** [comment]
Hi @DarioRomano,
sorry for the late response. The packages considering the metamodel (all packages with the prefix de.vill.*) are no longer part of the parser, but can be found here: https://github.com/Universal-Variability-Language/java-fm-metamodel
So you need to include the following Maven dependency to use those.
```
<dependency>
    <groupId>io.github.universal-variability-language</groupId>
    <artifactId>uvl-metamodel</artifactId>
    <version>1.0</version>
</dependency>
```

**Comment by DarioRomano** [comment]
> The packages considering the metamodel (all packages with the prefix de.vill.*) are no longer part of the parser, but can be found here: https://github.com/Universal-Variability-Language/java-fm-metamodel

Thanks, Kevin has already pointed me that way. It would be cool if that information could be found somewhere in the readme, as the landing page for the repo still mentions all the methods for the deprecated maven build, and the java usage section mentions classes which are not part of this repo, but the one you mentioned above.

I'm still kind of stuck during runtime, because when trying to run the parser, and having imported ANTLR4 runtime version 4.13.1 I get an error that the source was compiled with version 4.8, but when I use 4.8 it says it was compiled with 4.13.1. Any idea how I can match up the ANTLR dependencies, so the parser works?

**Comment by SundermannC** [comment]
I agree that we need to update the documentation as soon as possible to match the new structure. I created an issue #42  for that.

I have not had this problem yet, but I will try to reproduce it and get back to you. 
@jagalindo Maybe you already have an idea?

**Comment by gravemalte** [comment]
Closed due to the restructuring in https://github.com/Universal-Variability-Language/uvl-parser/commit/caa81489c683cd72adb2f9f5238b50aa831230db
